TWLight/static/css/new-local.rtl.css

```diff
@@ -189,6 +189,292 @@ LOGIN PARTIAL CSS
     padding: 0px 10px 0px 0px;
 }
 
+/*
+----------------------------------------------------------------------------
+HOMEPAGE CAROUSEL CSS
+--------------------------------------------------------------------------
+*/
```

Comment:
This file seems out of date. Can we just delete and regenerate with our cssjanus script?

Reply:
I will make the changes manually.

Reply:
This class has been updated! Let me know if I missed something!